p22/src/calc.rs

```diff
@@ -0,0 +1,104 @@
+// Conversion from celsius to  farenheit:
+/// Given a value in celsius, return the corresponding value in farenheit
+///
+/// # Example
+///
+/// ```
+/// assert_eq!(104, p22::calc::celsius2farenheit(40));
+/// ```
+
+pub fn celsius2farenheit(celsius: i32) -> i32 {
+    let farenheit: i32 = (celsius * 9 / 5) + 32;
+    farenheit
```

Comment:
You do not need temporary variable here and in `farenheit2celsius`. You can write:
```rust
ub fn celsius2farenheit(celsius: i32) -> i32 {
    (celsius * 9 / 5) + 32
}
```